frappe/contacts/doctype/address/address.py

```diff
@@ -204,25 +197,6 @@ def get_address_templates(address):
 	else:
 		return result
 
-@frappe.whitelist()
-def get_shipping_address(company, address = None):
-	filters = [
-		["Dynamic Link", "link_doctype", "=", "Company"],
-		["Dynamic Link", "link_name", "=", company],
-		["Address", "is_your_company_address", "=", 1]
-	]
-	fields = ["*"]
-	if address and frappe.db.get_value('Dynamic Link',
-		{'parent': address, 'link_name': company}):
-		filters.append(["Address", "name", "=", address])
-
-	address = frappe.get_all("Address", filters=filters, fields=fields) or {}
-
-	if address:
-		address_as_dict = address[0]
-		name, address_template = get_address_templates(address_as_dict)
-		return address_as_dict.get("name"), frappe.render_template(address_template, address_as_dict)
-
 def get_company_address(company):
```

Comment:
What about this? Considering that `Company` is something to be associated with ERPNext..

Reply:
Yeah, this should be moved to `erpnext` as it is imported by `erpnext` code only. In a separate PR maybe.

Reply:
I'll separate this in another PR.
There is plenty of other code like this will do it stepwise.